src/Configuration.php

```diff
@@ -12,6 +12,7 @@
 namespace LivingMarkup;
 
 use Symfony\Component\Yaml\Yaml;
+use Symfony\Component\Yaml\Parser;
```

Comment:
I'm going to take your expertise that the Symfony components performing better YAML parsing, which justifies reliance on them in core.